start.php

```diff
@@ -54,8 +53,8 @@ function mentions_get_views() {
  * @return string
  */
 function mentions_rewrite($hook, $entity_type, $returnvalue, $params) {
-
 	$regexp = mentions_get_regex();
+
```

Comment:
Please avoid whitespace changes what are not related to the actual functional changes.